packages/json-file/src/JsonFile.ts

```diff
@@ -212,7 +212,7 @@ async function writeAsync<TJSONObject extends JSONObject>(
   options?: Options<TJSONObject>
 ): Promise<TJSONObject> {
   if (options?.ensureDir) {
-    await mkdirp(path.dirname(file));
+    await fs.promises.mkdir(path.dirname(file), { recursive: true });
```

Comment:
`ensureDir` ?

Reply:
Nah, that's an alias that only exists in `fs-extra`. I want [`fs.promises.mkdir`](https://nodejs.org/api/fs.html#fs_fspromises_mkdir_path_options), so `fs-extra` is no longer needed.